sirepo/smtp.py

```diff
@@ -94,14 +95,14 @@ def _init():
         _cfg.server = "not " + _DEV_SMTP_SERVER
         _SEND = _send_directly
         return
-    _SEND = _send_with_auth
+    _SEND = _send_to_configured_smtp_server
     if pkconfig.in_dev_mode():
         if _cfg.server is None:
             _cfg.server = _DEV_SMTP_SERVER
         return
-    if _cfg.server is None or _cfg.user is None or _cfg.password is None:
+    if _cfg.server is None:
```

Comment:
It should probably be an error if user is set without password. Certainly password set without user.  Catch that at config time.

Reply:
8f9c96785